.github/workflows/prerelease.yml

```
@@ -80,6 +78,14 @@ jobs:
           sudo mv /usr/bin/python3 /usr/bin/python3-default
           sudo ln -s ${{ env.pythonLocation }}/bin/python3 \
           /usr/bin/python3
+      - name: Install process user data file script dependencies
+        run: python -m pip install boto3 jinja2
+      - name: Process user data file
+        id: process-user-data
+        # Process the user data script with the Windows admin password
+        # pulled from AWS Parameter Store. Return the admin password to
+        # pass into Packer via command line variable.
```

Same comment update as `build.yml`.

```suggestion
        # Process the user data script to set the Windows Administrator
        # password to the value retrieved from AWS Parameter Store. Set a step
        # output to use the retrieved password as the winrm_password value
        # with Packer via command line argument.
```

---

@mcdonnnj resolved with [e90b46c](https://github.com/cisagov/windows-server-packer/pull/10/commits/e90b46c476283f3e0c54ea56cc6c551d28a44ab5)